lib/ci-stack.ts

```diff
@@ -34,9 +34,17 @@ export class CIStack extends Stack {
     });
 
     const useSslParameter = this.node.tryGetContext('useSsl');
+    if (useSslParameter !== 'true' && useSslParameter !== 'false') {
+      throw new ReferenceError('useSsl parameter is required to be set as "true" or "false"');
+    }
+
     const useSsl = useSslParameter === 'true';
 
     const runWithOidcParameter = this.node.tryGetContext('runWithOidc');
+    if (runWithOidcParameter !== 'true' && runWithOidcParameter !== 'false') {
+      throw new ReferenceError('runWithOidc parameter is required to be set as "true" or "false"');
```

Comment:
Why use `ReferenceError`, just make this an `Error` type unless there is something like an argument error

Reply:
I actually used `Error` before and then later changed to `ReferenceError` since value of the variable `runWithOidcParameter` and  `useSslParameter` should only be true or false. Any other value cannot be assigned to it and I think should cause a referencing error as it should "technically" be undefined.

That being said, there isn't any rule to this that I could find. So I can put whatever everyone agrees to correct. Thoughts?

Reply:
Found it https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError

Doesn't seem correct, just use `Error` the intent isn't that someone catches it and you included the needed details so it can be remediated